CHANGELOG.next.toml

```diff
@@ -10,6 +10,18 @@
 # references = ["smithy-rs#920"]
 # meta = { "breaking" = false, "tada" = false, "bug" = false }
 # author = "rcoh"
+[[aws-sdk-rust]]
+message = "Runtime crates (other than aws-config) no longer have default features. You must now specify the features that you want when you add a dependency to your `Cargo.toml`."
```

Comment:
I think this is a breaking change from the perspective of smithy-rs, but not necessarily from the perspective of aws-sdk-rust (depending on which crates customers were depending on). For customers using just `aws-config` and some service packages, they shouldn't notice a change at all. I think we should nuance the language a little.

Reply:
If someone was directly using something like `aws-sigv4` then they'd be affected by this change. I don't think that that'll be an issue in practice but it is a breaking change. That being said, I'm for setting whatever expectations you think are the most helpful. I'll bother you about it tomorrow.

Reply:
Yeah, I would probably mark it as a breaking change, but preface the message with something like, "If you directly depend on AWS or Smithy runtime crates (e.g., AWS crates not named `aws-config` or prefixed with `aws-sdk-`), ...".